### ISSUE TITLE
Comparison operators "Equals column" or "Differs to column" fail

### ISSUE THREAD

**Issue by leehart** [cggh/panoptes#921]
Samples_and_Variants > Variants > Add Filter
Add Criterion > {Variant name = SNP_00001}
Change the comparison operator to "Equals column" or "Differs to column".
See that the option is not selected.
See an error in the JS console:
```
Uncaught TypeError: groups.map is not a function
    at otherColumnSelect (QueryEditor.js:472)
    at Object.render (QueryEditor.js:610)
```

**Comment by leehart** [comment]
I expect this will be fixed via #920

**Comment by leehart** [comment]
Looks like a simple typo
```
groups.map((group) => {...
```
fixed by
```
_map(groups, (group) => {...
```
